src/engine/renderer/tr_image.cpp

```diff
@@ -1682,57 +1734,24 @@ static void R_LoadImage( const char **buffer, byte **pic, int *width, int *heigh
 		// a loader was found
 		if ( i < numImageLoaders )
 		{
-			if ( *pic == nullptr )
-			{
-				// loader failed, most likely because the file isn't there;
-				// try again without the extension
-				COM_StripExtension3( token, filename, MAX_QPATH );
-			}
-			else
+			if ( *pic != nullptr )
 			{
 				// something loaded
 				return;
 			}
 		}
 	}
 
-	int bestLoader = -1;
-	const FS::PakInfo* bestPak = nullptr;
-
-	// Darkplaces or Doom3 packages can ship alternative texture path in the form of
-	//   dds/<path without ext>.dds
-	std::string altName = Str::Format("dds/%s.dds", filename);
-	bestPak = FS::PakPath::LocateFile(altName);
-
-	// If this alternative path exists, it's expected to be loaded as the best one
-	// except when it goes against Daemon's rule to load the hardcoded one if exists
-	// because this dds alternative is only supported for compatibility with
-	// third-party content
-	if ( bestPak != nullptr ) {
-		LoadDDS( altName.c_str(), pic, width, height, numLayers, numMips, bits, alphaByte );
-		return;
-	}
-
-	// try and find a suitable match using all the image formats supported
-	// prioritize with the pak priority
-	for ( i = 0; i < numImageLoaders; i++ )
-	{
-		std::string altName = Str::Format("%s.%s", filename, imageLoaders[i].ext);
-		const FS::PakInfo* pak = FS::PakPath::LocateFile(altName);
+	// loader failed, most likely because the file isn't there;
```

Comment:
The comment doesn't make sense here since this path can be reached without a loader failing.